swupd/helpers_test.go

```diff
@@ -218,3 +200,37 @@ func mustCreateManifests(t *testing.T, ver uint32, minVer bool, format uint, tes
 		t.Fatal(err)
 	}
 }
+
+func checkManifestContains(t *testing.T, testDir, ver, name string, subs []string, expRes bool) {
```

Comment:
Optional: assuming you drop the bool, you can use varargs to allow both single string and list of strings.

    func checkManifestContains(..., subs ...string) {
        // can use subs as a []string, so implementation don't change.
    }

    // Then we can call:
    checkManifestContains(t, dir, "10", "MoM", "alalallaa")
    // or
    mySubs := []string{...}
    checkManifestContains(t, dir, "10", "MoM", mySubs...)

Reply:
Even if we don't drop the bool (which I strongly support), we could move the `expRes` before `subs`.